### PR TITLE
NAS-107694 / 12.0 / Add Pool options dialog with auto-trim option

### PR DESCRIPTION
**Pull request by erinix**
NAS-107694

### CODE REVIEW

File: src/app/helptext/storage/volumes/volume-list.ts [hunk 1]
```diff
@@ -125,6 +125,18 @@ encryption_options_dialog: {
     confirm_checkbox: T('Confirm'),
     save_button: T('Save')
 },
+pool_options_dialog: {
+    dialog_title: T('Edit Pool Options for '),
+    autotrim_placeholder:T('Auto TRIM'),
+    autotrim_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/storage/volumes/volume-list.ts:131]: Help text suggestion:
`Enable for TrueNAS to periodically review data blocks and identify empty blocks of obsolete blocks that can be deleted. Unset to incorporate day block overwrites when a device write is started (default).

Comment (Mrt134) [inline src/app/helptext/storage/volumes/volume-list.ts:131]: Help text suggestion:
`Enable for TrueNAS to periodically review data blocks and identify empty blocks of obsolete blocks that can be deleted. Unset to incorporate day block overwrites when a device write is started (default).

**Review by Mrt134 — APPROVED** [review]
I added a rough help text suggestion for auto trim, but needs a sanity check.

**Review by dkmullen — APPROVED** [review]
lgtm

**Pull request merged**